src/CLI/actioner/sonic-cli-ifa.py

```diff
@@ -132,11 +132,42 @@ def get_tam_ifa_status(args):
 
     show_cli_output("show_tam_ifa_status.j2", api_response)
 
+def get_tam_ifa_flow_stats(args):
+    api_response = {}
+    api = cc.ApiClient()
+
+    # connect to COUNTERS_DB
+    counters_db = ConfigDBConnector()
+    counters_db.db_connect('COUNTERS_DB')
+
+    if len(args) == 0:
+       path = cc.Path('/restconf/data/sonic-ifa:sonic-ifa/TAM_INT_IFA_FLOW_TABLE')
+    else:
+       path = cc.Path('/restconf/data/sonic-ifa:sonic-ifa/TAM_INT_IFA_FLOW_TABLE/TAM_INT_IFA_FLOW_TABLE_LIST={name}', name=args[0])
+
+    response = api.get(path)
+
+    if response.ok():
+        if response.content:
+            if len(args) == 0:
+                api_response = response.content['sonic-ifa:TAM_INT_IFA_FLOW_TABLE']['TAM_INT_IFA_FLOW_TABLE_LIST']
+            else:
+                api_response = response.content['sonic-ifa:TAM_INT_IFA_FLOW_TABLE_LIST']
+            for i in range(len(api_response)):
+                acl_counter_key = 'COUNTERS:' + api_response[i]['acl-table-name'] + ':' + api_response[i]['acl-rule-name']
+                flow_stats = counters_db.get_all(counters_db.COUNTERS_DB, acl_counter_key)
```

Comment:
Should we use data from Redis-DB? Can't we use REST for the same ?

Reply:
How the user will be able to get statistics data for REST?

Reply:
@asha-behera Right now the support is only for CLI. 
As per @PrabhuSreenivasan the counters are available (looks to be only available) at counters DB. I dont see counters in sonic-acl.yang.
I could define a counters container in ifa yang but who is going to populate them?

Reply:
@spenugondaa : stats info should be "state" ( config=false)  information and not config information.

Reply:
@asha-behera @PrabhuSreenivasan please take a look. I am using now oc acl yang to fetch the counters.